main.py

```diff
@@ -1,10 +1,46 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+    for player_nickname, player_info in players.items():
+        email = player_info.get("email")
+        bio = player_info.get("bio")
+        race_ = player_info.get("race")
+        skills_ = race_.get("skills")
+        guild_ = player_info.get("guild")
+
+        race = Race.objects.get_or_create(
+            name=race_.get("name"),
+            description=race_.get("description")
+        )[0]
```

Comment:
```suggestion
        race, _ = Race.objects.get_or_create(
            name=race_.get("name"),
            description=race_.get("description")
        )
```
Better to do like this)
Fix in all places

Reply:
Bu why its better?

Reply:
It is more clear code. If you know that it will return tuple with two params better to use: 
variable, _
In all future project you will use this construction instead of [0]
[0] you can use if there unknown count of parameters